xonsh/built_ins.py

```diff
@@ -714,6 +715,7 @@ def load_builtins(execer=None, config=None, login=False, ctx=None):
     builtins.__xonsh_ensure_list_of_strs__ = ensure_list_of_strs
     builtins.__xonsh_list_of_strs_or_callables__ = list_of_strs_or_callables
     builtins.__xonsh_completers__ = xonsh.completers.init.default_completers()
+    builtins.__xonsh_events__ = events
```

Comment:
There's a commit missing. I could have sworn I changed the name and initialized it in the module (so that core could load it).